users/models.py

```diff
@@ -62,6 +62,7 @@ class User(AbstractBaseUser):
     description = models.TextField(default='', blank=True)
     is_active = models.BooleanField(default=False)
     confirm_token = models.CharField(default=create_hash, max_length=40)
+    password_reset_token = models.CharField(default=create_hash, max_length=40)
```

Comment:
This was probably not well documentend, but the `password_token` is exactly what you would have to use. You can delete this line, or the other one.